docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.ts

```diff
@@ -34,6 +34,8 @@ export class CssUxIconsComponent {
 
     uxIconComponentRoute: string;
 
+    size: string = '24px';
```

Comment:
I wonder here is 24px the best default here? Is this going to be the most commonly used size in an application?

I realise that the icons naturally are 24x24 in size and look best at this resolution, but now we have the copy and paste feature, by default if a consumer opens the popover and clicks the copy icon it will copy a 24px size icon. Is this likely to be the most common choice?

If not then we should consider the 16px variation as the default so they either don't have to switch sizes to get the correct sample code or manually have to edit the code after they copy and paste it in to their application.

Thoughts?

Reply:
The size has now been removed, from the code snippet. I do agree with this as it nearly gives the impression that 24px is the default size, without having to add extra code.

@donna-marijne